### DISCUSSION TITLE
可否实现根据注释对应的颜色为注释添加相应的标签？

### DISCUSSION THREAD

**Discussion by Dobby2046** [windingwind/zotero-actions-tags#281] [Ideas]
我在阅读文献时根据用不同的颜色标记了文章中各个结构的文本，我想让zotero对不同颜色的文本添加标签，方便我在obsidian中自动根据标签总结论文。

**Comment by ulahcherubim** [comment]
See this discussion in Obsidian forums: https://forum.obsidian.md/t/zotero-integration-import-templates/36310
Also note that you can add template variables in Zotero to get them under certain headers and such.  
https://www.zotero.org/support/note_templates
https://www.zotero.org/support/pdf_reader

**Reply by windingwind** [reply]
Thanks for the comment. I don't mean to discourage your participation here, but this is more an *alternative* rather than a *solution* to what the OP asked for.

I would still appreciate any contribution to creating a script for this request, which should not be difficult.

**Reply by ulahcherubim** [reply]
Thanks for the clarification. I don't have much knowledge about programming, so that's my maximum capacity.

**Comment by poricaihong** [comment]
好像科研少女王同学搞了个笔记流实现了，求具体代码。

**Comment by appwcn** [comment]
https://github.com/windingwind/zotero-actions-tags/discussions/339
